api/types.go

```diff
@@ -103,6 +103,7 @@ type Options struct {
 	MirostatEta      float32  `json:"mirostat_eta,omitempty"`
 	PenalizeNewline  bool     `json:"penalize_newline,omitempty"`
 	Stop             []string `json:"stop,omitempty"`
+	Cache            bool     `json:"cache,omitempty"`
```

Comment:
There's a `DefaultOptions()` function further down in this file, let's add `Cache: true` to that.

```
func DefaultOptions() Options {
	return Options{
		// options set on request to runner
		// other stuff that is already here ...
		Seed:             -1,
                Cache: true

		Runner: Runner{
			// options set when the model is loaded
			// etc ...
		},
	}
}
```